docs/get.md

```diff
@@ -12,3 +13,63 @@ This command will display the recovery-profile value set for the osd.
 $ odf get recovery-profile
 # high_recovery_ops
 ```
+
+## health
+
+Health command check health of the cluster and common configuration issues. Health command currently validates these things configurations (let us know if you would like to add other validation in health command):
+
+1. at least three mon pods should running on different nodes
+2. mon quorum and ceph health details
+3. at least three osd pods should running on different nodes
+4. all pods 'Running' status
+5. placement group status
+6. at least one mgr pod is running
+
+Health commands logs have three ways of logging:
```

Comment:
Lines 28-32 seem like they are not necessary to describe, perhaps remove them from the doc to keep it simple?

Reply:
yeah, it makes sense to remove. Done